doc/configuration.md

```diff
@@ -1,12 +1,14 @@
 # Configuration
 
+This is the configuration to use for [event subscribers](https://github.com/tango-controls-hdbpp/hdbpp-es) to use this backend
+
 ## Library Configuration Parameters
 
 Configuration parameters are as follows:
 
 | Parameter | Mandatory | Default | Description |
 |------|-----|-----|-----|
-| libname | true | None | Must be "libhdb++timescale.so" |
+| libname | true | None | Must be "libhdb++timescale.so", this parameter is getting deprecated as the backend is enforced during compilation |
 | connect_string | true | None | Postgres connection string, eg user=postgres host=localhost port=5432 dbname=hdb password=password |
```

Comment:
Description needs to be adjusted since the possibility of dynamic loading of the library remains.

Reply:
thank you, I changed it.